src/Modules/Trinity.FFI/Trinity.FFI/TrinityWrapper.cs

```diff
@@ -159,6 +160,36 @@ public static TrinityErrorCode trinity_ffi_local_savecell_2(long cellId, CellAcc
                 return TrinityErrorCode.E_FAILURE;
             }
         }
+
+        public static TrinityErrorCode trinity_ffi_local_savecell_3(IntPtr cell)
+        {
+            try
+            {
+                ICell c = (ICell)GCHandle.FromIntPtr(cell).Target;
```

Comment:
might help to further improve performance if we do ObjectStore right from here :) Remember, our goal is to minimize cross-runtime calls and GCHandle. So it does not make a lot of sense to go back and forth between a foreign runtime and clr, to put an object into the cache